src/test/kotlin/com/example/realworldkotlinspringbootjdbc/api_integration/CommentTest.kt

```diff
@@ -63,36 +67,47 @@ class CommentTest {
 
             /**
              * then:
+             * - ステータスコードが 200
              * - 作成日時と更新日時以外を比較する
              */
-            actual.andExpect { status { isOk() } }
-                .andExpect {
-                    content {
-                        json(
-                            """
-                                {
-                                  "comments": [
-                                    {
-                                      "id": 1,
-                                      "body": "dummy-comment-body-01",
-                                      "authorId": 3
-                                    },
-                                    {
-                                      "id": 3,
-                                      "body": "dummy-comment-body-03",
-                                      "authorId": 2
-                                    },
-                                    {
-                                      "id": 5,
-                                      "body": "dummy-comment-body-02",
-                                      "authorId": 3
-                                    }
-                                  ]
-                                }
-                            """.trimIndent()
-                        )
+            val expected =
+                """
+                    {
+                      "comments": [
+                        {
+                          "id": 1,
+                          "body": "dummy-comment-body-01",
+                          "createdAt": "2022-09-26T15:00:00.000Z",
+                          "updatedAt": "2022-09-26T15:00:00.000Z",
+                          "authorId": 3
+                        },
+                        {
+                          "id": 3,
+                          "body": "dummy-comment-body-03",
+                          "createdAt": "2022-09-26T15:00:00.000Z",
+                          "updatedAt": "2022-09-26T15:00:00.000Z",
+                          "authorId": 2
+                        },
+                        {
+                          "id": 5,
+                          "body": "dummy-comment-body-02",
+                          "createdAt": "2022-09-26T15:00:00.000Z",
+                          "updatedAt": "2022-09-26T15:00:00.000Z",
+                          "authorId": 3
+                        }
+                      ]
                     }
-                }
+                """.trimIndent()
+            val actualResponseBody = actual.andExpect { status { isOk() } }.andReturn().response.contentAsString
+            JSONAssert.assertEquals(
+                expected,
+                actualResponseBody,
+                CustomComparator(
+                    JSONCompareMode.STRICT,
+                    Customization("*.createdAt") { _, _ -> true },
+                    Customization("*.updatedAt") { _, _ -> true },
```

Comment:
[imo]

ちょっとやりすぎにもなるかもしれませんが、以下なんてどうでしょう？

やっていること
- ISO8601形式であること（形は色々ありますが、特定の形式を期待）
- TimeZoneがUTCであること（目的：TimeZoneの期待）
- parseできること（目的：時刻として矛盾がないか確認）

（毎度正規表現が入るのも微妙なので）検証メソッドを `別に` 切り出しても良いかもです（例：VerificationHelper.ktのcompanion object）

```suggestion
            val expectIso8601UtcAndParseable = { o: Any ->
                assertThat(o.toString())
                    .`as`("YYYY-MM-DDTHH:mm:ss.SSSXXX(ISO8601形式)で、TimeZoneはUTCである")
                    .matches("([0-9]{4})-([0-9]{2})-([0-9]{2}T([0-9]{2}):([0-9]{2}):([0-9]{2}).([0-9]{3})Z)")
                kotlin.runCatching { OffsetDateTime.parse(o.toString()) }.isSuccess
            }
            JSONAssert.assertEquals(
                expected,
                actualResponseBody,
                CustomComparator(
                    JSONCompareMode.STRICT,
                    Customization("*.createdAt") { actualCreatedAt, _ -> expectIso8601UtcAndParseable(actualCreatedAt) },
                    Customization("*.updatedAt") { actualUpdatedAt, _ -> expectIso8601UtcAndParseable(actualUpdatedAt) },
                )
            )
```